activitypub.php

```diff
@@ -120,12 +121,12 @@ function plugin_settings_link( $actions ) {
  */
 function add_rewrite_rules() {
 	if ( ! \class_exists( 'Webfinger' ) ) {
-		\add_rewrite_rule( '^.well-known/webfinger', 'index.php?rest_route=/activitypub/1.0/webfinger', 'top' );
+		\add_rewrite_rule( '^.well-known/webfinger', 'index.php?rest_route=/' . ACTIVITYPUB_REST_NAMESPACE . '/webfinger', 'top' );
```

Comment:
Does this even work if .com blocks the local API endpoint or do we have to rethink this completely?

Reply:
So it's the `/wp-json/` route that's blocked but the underlying `index.php?rest_route=*` probably still works fine. (Checked now and nope, not fine in initial testing)

But the `add_rewrite_call` itself gets handled uniquely on wpcom so more investigation will be required, and if we need to get help from systems to allow any calls within `.well-known`.

Reply:
Maybe we have to simply use a different route, not under the API endpoint. Something like: 

`\add_rewrite_rule( '^.well-known/webfinger', 'index.php?webfinger=endpoint', 'top' );`

Reply:
I investigated a bit more and we're almost certainly going to need help from the systems folks to get anything handled under the `.well-known` namespace. I can't even put a PHP file at `.well-known/webfinger/index.php` and reach it. It's being directed elsewhere at the nginx level, so we'll have to ask. I assume it has to be exactly of the pattern `/.well-known/webfinger` for spec conformity?

I also see `nodeinfo` and `x-nodeinfo2`, anything else that needs to be in `.well-known`? And my impression was that only `webfinger` was _required_ and can't be changed to another URL pattern right?

Reply:
only WebFinger and NodeInfo for the moment.